examples/inlinebot.py

```diff
@@ -44,7 +45,7 @@ def help(bot, update):
 
 
 def escape_markdown(text):
```

Comment:
Can we replace this as well? We have `telegram.utils.helpers.escape_markdown` now.